modules/nf-core/mmseqs/tsv2exprofiledb/main.nf

```diff
@@ -0,0 +1,45 @@
+process MMSEQS_TSV2EXPROFILEDB {
+    tag "$db"
+    label 'process_high'
+
+    conda (params.enable_conda ? "bioconda::mmseqs2=14.7e284" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mmseqs2:14.7e284--pl5321hf1761c0_0':
+        'quay.io/biocontainers/mmseqs2:14.7e284--pl5321hf1761c0_0' }"
+
+    input:
+    path db
```

Comment:
I don't think meta in this module is very useful, we don't use it currently in proteinfold and can not foresee scenarios where it could be useful, AFAIK is not mandatory to provide a meta, is it?

Reply:
Nope indeed, I was just curious why